x-pack/plugins/alerting/common/alert.ts

```diff
@@ -135,3 +136,14 @@ export interface ActionVariable {
   deprecated?: boolean;
   useWithTripleBracesInTemplates?: boolean;
 }
+
+export interface RuleMonitoring extends SavedObjectAttributes {
+  execution: {
+    history: Array<{
+      success: boolean;
```

Comment:
I think we can add the timestamp here because it should be the same one between success and duration recording. Am I correct here?

Reply:
Yes we can easily add it here!